packages/grid/_modules_/grid/locales/bgBG.ts

```diff
@@ -6,6 +6,7 @@ const bgBGGrid: Partial<GridLocaleText> = {
   // Root
   rootGridLabel: 'мрежа',
   noRowsLabel: 'Няма редове',
+  // noResultsOverlayLabel: 'No results found.',
```

Comment:
this is going to throw an exception in this language

Reply:
Why? Or even better, do you have a proof? It's not the case. Proof: https://material-ui.com/components/data-grid/localization/#translation-keys.

Reply:
It won't. We use the English language as fallback:

https://github.com/mui-org/material-ui-x/blob/4bd386579466b6efd7a1029097a43f1b3b7ca178/packages/grid/_modules_/grid/hooks/utils/useOptionsProp.ts#L43-L50